urgent-care/src/test/java/gov/va/api/health/urgentcare/service/controller/BundlerTest.java

```diff
@@ -0,0 +1,103 @@
+package gov.va.api.health.urgentcare.service.controller;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import gov.va.api.health.urgentcare.api.bundle.AbstractBundle;
+import gov.va.api.health.urgentcare.api.bundle.AbstractBundle.BundleType;
+import gov.va.api.health.urgentcare.api.bundle.AbstractEntry;
+import gov.va.api.health.urgentcare.api.bundle.BundleLink;
+import gov.va.api.health.urgentcare.api.bundle.BundleLink.LinkRelation;
+import gov.va.api.health.urgentcare.api.elements.Meta;
+import gov.va.api.health.urgentcare.api.resources.Resource;
+import gov.va.api.health.urgentcare.service.controller.Bundler.BundleContext;
+import gov.va.api.health.urgentcare.service.controller.PageLinks.LinkConfig;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Function;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Value;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+@SuppressWarnings("WeakerAccess")
+public class BundlerTest {
+  private static final Function<FugaziCdwRoot, FugaziArgo> FUGAZIMUS_PRIME =
```

Comment:
I'm fine keeping Fugazi, but I think we should change 'Argo' as this isn't Urgonaut.